packages/keycloack/src/service/validate-acess-token.service.ts

```diff
@@ -0,0 +1,13 @@
+import { Injectable } from '@nestjs/common'
+import { KeycloakClient } from '../client'
+
+@Injectable()
+export class ValidateAccessToken {
```

Comment:
What do you think of adding service to class name?
```suggestion
export class ValidateAccessTokenService {
```